.env.example

```
@@ -1,2 +1,16 @@
-DATABASE_URL="mongodb://localhost:27017/dev"
-JWT_SECRET="4gFk^2-4]DrX8xEjvis{GMm[tM3W(y"
+# Environment
+NODE_ENV=development
+
+# Server
+PORT=3000
+HOST=localhost
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The HOST key should go before the PORT key

(UnorderedKey)